dataland-frontend/src/components/wrapper/AuthorizationWrapper.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <div v-if="hasUserRequiredRole">
+  <div v-if="hasUserRequiredRole || isUserDataOwner">
     <slot></slot>
   </div>
```

Comment:
What about api key authentication?
If I am a data owner for company XYZ and I request an API key, then I should probably expect to be able to upload data for company XYZ by using an API-key, right?

How do we handle this? Do we ignore it? ADjust it some time else?  Never adjust it?  What is the plan here.

Reply:
Would think so. However i think that is its own story, because i think the apikey manager has to be updated for this, as currently i think the apikey manager does not care for dataownership

Reply:
Habe gerade nochmal in den Quellcode geguckt und so wie ich es verstehe, sollte das bereits automatisch funktionieren.

D.h. wenn du als DATA_READER einen api-key holst, dann anschließend data-owner von einer company wirst, und dann versuchst etwas für diese company hochzuladen (über die API), sollte es funktionieren.

Kannst du das vllt einmal ausprobieren?  Wenn du es bestätigen kannst, dann schließe ich die conversation.

Reply:
Scheint so. 
Habe mich mit dme ApiKey hierrüber angemeldet:
![image](https://github.com/d-fine/Dataland/assets/104447806/542b9545-2e12-4e67-a92d-ef3649b2254a)
Und konnte dann für die zugewiesenen Companys Daten hochladen.

Reply:
Perfekt. Dann schließe ich